Dockerfile.in

```diff
@@ -1,15 +1,15 @@
 FROM UPSTREAM_REPO
 
+MAINTAINER Randy Fay <rfay@drud.com>
```

Comment:
As a general rule I don't think we've been using maintainer lines. If we do, perhaps it should be "team@drud.com"?